package.json

```diff
@@ -84,14 +89,23 @@
       "stylelint-no-unsupported-browser-features"
     ],
     "rules": {
+      "string-quotes": "single",
+      "color-hex-length": "long",
+      "function-parentheses-space-inside": "always",
+      "function-url-quotes": "always",
       "at-rule-empty-line-before": [
         "always",
         {
           "except": [
-            "first-nested"
+            "first-nested",
+            "blockless-after-same-name-blockless"
```

Comment:
Prevents errors with blockless `@rules` that follow the same name (e.g. `@import`, `@include`).